tests/integration/components/project-recommendation-new-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | project-recommendation-new', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`{{project-recommendation-new}}`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      {{#project-recommendation-new}}
+        template block text
+      {{/project-recommendation-new}}
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
```

Comment:
This should be a real test...

Reply:
Maybe I'll add some unit tests for saving/creating Recommendation instances, and leave the integration test for PR/Issue 533 (Recommendation form template)?